### DISCUSSION TITLE
8K清晰度下载支持

### DISCUSSION THREAD

**Discussion by aliofwind** [the1812/Bilibili-Evolved#2589] [功能建议 / Ideas]
B站今天开放了8K清晰度
https://www.bilibili.com/read/cv14296511
目前测试情况是Safari的UA可以获取到8K这一档，quality为127，只有HEVC编码

**Comment by the1812** [comment]
有请求的 URL 链接吗, 类似这样的: `https://api.bilibili.com/x/player/playurl?avid=764531297&cid=452674905&qn=120&otype=json&fourk=1&fnver=0&fnval=80`

**Reply by Q16KBreak** [reply]
https://api.bilibili.com/x/player/playurl?cid=455439756&bvid=BV1qM4y1w716&qn=112&type=&otype=json&fourk=1&fnver=0&fnval=2000&session=d75dddb93b260100fd4773800868827a
这算不算

**Reply by the1812** [reply]
算

**Reply by Q16KBreak** [reply]
https://api.bilibili.com/x/player/playurl?cid=455022263&bvid=BV1KS4y197BN&qn=112&type=&otype=json&fourk=1&fnver=0&fnval=2000&session=30e88fb8ab116456a09058bafb40bfd8
https://api.bilibili.com/x/player/playurl?cid=452414425&bvid=BV11f4y1K7Wx&qn=120&type=&otype=json&fourk=1&fnver=0&fnval=2000&session=a42634c161d834fd96cc27ad7fae5c7d
https://api.bilibili.com/x/player/playurl?cid=452039141&bvid=BV1W34y1R7G3&qn=112&type=&otype=json&fourk=1&fnver=0&fnval=2000&session=8ba5e20a69f29fa9127bb665e3b4c263

**Comment by xincheng213618** [comment]
测试了一下不需要 Safari 的UA,chrome的也可以，和4K相比主要的更改为fnval=80改成fnval=2000

**Comment by xincheng213618** [comment]
相关改动参照https://github.com/wolfbolin/BiliUtil/issues/46
